sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -332,6 +334,10 @@ class ShareProperties(DictMixin):
     :ivar int remaining_retention_days:
         To indicate how many remaining days the deleted share will be kept.
         This is a service returned value, and the value will be set when list shared including deleted ones.
+    :ivar ~azure.storage.fileshare.models.ShareRootSquash or str root_squash:
+        Possible values include: 'NoRootSquash', 'RootSquash', 'AllSquash'.
+    :ivar ~azure.storage.fileshare.models.ShareProtocols or str protocols:
```

Comment:
type should be a list

Reply:
It shouldn't be a list since the service returns a string of enabled protocols and we will deserialize that into a list for the user.

Reply:
It shouldn't since the service returns a string of protocols and deserialize it into a list in `_models.py` here https://github.com/Azure/azure-sdk-for-python/blob/4592d8537d5d07d6ea667b6530f2b2c28993d30e/sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py#L360